girder/models/model_base.py

```diff
@@ -45,6 +45,37 @@
 _modelSingletons = []
 
 
+def _permissionClauses(user=None, level=None, prefix=''):
```

Comment:
Is there someone downstream who we know will use this `prefix` feature? It doesn't look like we're using it in this repo.

Reply:
The acl_mixin uses a prefix of `'_access.'`.